java/ql/integration-tests/java/javac-tool-custom-file/test.expected

```diff
@@ -0,0 +1 @@
+| Main |
```

Comment:
Should this get some kind of location based on `https://nonesuch.imaginary/somedir/Main.java`?

Reply:
Currently the loc seen will be inside the db's scratch dir where the temporary is created. Almost necessarily the loc given by toURI or similar is going to be something vscode etc won't understand, like a jar-relative URI or the like; at least this location will be clickable, though considered outside the source-root. I haven't exposed it in this test because it isn't stable from test run to run (contains a temporary dirname).

I suggest we don't tackle this one at this juncture, since I'm not clear how we can give it a more useful location that makes sense to VSCode / another client, or the user.